Terraform/Home_Task_2/modules/alb/main.tf

```diff
@@ -49,13 +39,36 @@ resource "aws_alb_listener" "listener_http" {
     target_group_arn = aws_alb_target_group.this_tg.arn
     type             = "forward"
   }
-} /*
-resource "aws_lb_target_group_attachment" "ec2_attach" {
-  target_group_arn = aws_alb_target_group.this_tg.arn
-  target_id        = aws_autoscaling_group.this_asg.id # what to set? alb_arn
-}*/
+}
 
 resource "aws_autoscaling_attachment" "asg_attachment_bar" {
   autoscaling_group_name = var.asg_id
   lb_target_group_arn    = aws_alb_target_group.this_tg.arn
+}
+
+resource "aws_security_group" "alb" {
+  name        = "terraform_alb_security_group"
+  description = "Terraform load balancer security group"
+  vpc_id      = var.vpc_id
+  /*
+  ingress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }*/
+
+  ingress {
+    from_port   = 80

```

Comment:
from_port, to_port parameters - define values of these parameters in modules/alb/variables.tf file (or use them if they already exist)